apps/src/music/player/soundSub.js

```diff
@@ -44,7 +38,7 @@ function createAudioContext(desiredSampleRate) {
   return context;
 }
 
-function WebAudio(bpm) {
+function WebAudio(delayTimeSeconds, releaseTimeSeconds) {
```

Comment:
Should we also use an `options` bundle here?

Reply:
Ah yes good call